src/main/java/Task.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Represents a task in the chatbot application.
+ * A task has a name, a completion status, and a unique task number.
+ */
+public class Task {
+    public String name; // The name or description of the task
+    public boolean hasDone; // The completion status of the task
+    public int taskNo; // The unique number assigned to the task
+
+    /**
+     * Constructs a new Task with the specified name and task number.
+     * Initially, the task is marked as not done.
+     *
+     * @param name The name or description of the task.
+     * @param taskNo The unique number assigned to the task.
+     */
+    public Task(String name, int taskNo) {
+        this.name = name;
+        this.hasDone=false;
```

Comment:
```suggestion
        this.hasDone = false;
```
just for consistency